packages/common/src/hardhat/tasks/collateralWhitelist.ts

```diff
@@ -10,6 +11,19 @@ import type { CombinedHRE } from "./types";
 
 dotenv.config();
 
+// LZ chainId mappings from https://github.com/LayerZero-Labs/wrapped-asset-bridge/blob/main/constants/chainIds.json
+const LZ_CHAIN_IDS: Record<number, number> = {
+  1: 101, // ethereum
+  10: 111, // optimism
+  137: 109, // polygon
+  1116: 153, // coredao
+  42161: 110, // arbitrum
+};
+
+const LZ_WRAPPED_TOKEN_BRIDGES: Record<number, string> = {
```

Comment:
Interesting: they use LZ as their canonical token bridge?

Reply:
yes, The Core Bridge is powered by LayerZero: https://docs.coredao.org/developer/develop-on-core/bridging-with-core/core-bridge-resources